packages/cdk/lib/stacks/core-stack.ts

```diff
@@ -7,6 +7,7 @@ import { Construct } from "constructs";
 import { PRODUCT_NAME, APP_NAME, VPC_PARAMETER_NAME } from "../constants";
 import { BucketDeployment, Source } from "aws-cdk-lib/aws-s3-deployment";
 import * as path from "path";
+import * as os from "os";
```

Comment:
how is it different from doing `import os from 'os';`?

Reply:
IDE shows there is no default export from 'os', which I thought they had, maybe this node version is different I am not sure. But I have now change to explicitly import only 'homedir'